src/applications/vaos/containers/ContactInfoPage.jsx

```diff
@@ -84,22 +106,26 @@ export class ContactInfoPage extends React.Component {
     const { schema, data, pageChangeInProgress } = this.props;
 
     return (
-      <SchemaForm
-        name="Contact info"
-        title="Contact info"
-        schema={schema || initialSchema}
-        uiSchema={uiSchema}
-        onSubmit={this.goForward}
-        onChange={newData =>
-          this.props.updateFormData(pageKey, uiSchema, newData)
-        }
-        data={data}
-      >
-        <FormButtons
-          onBack={this.goBack}
-          pageChangeInProgress={pageChangeInProgress}
-        />
-      </SchemaForm>
+      <div>
+        <h1 className="vads-u-font-size--h2">Contact information</h1>
+
+        <SchemaForm
```

Comment:
Huh. I think GH doesn't like the React fragment syntax; the syntax highlighting is all wonky now. :man_shrugging: 

Then again, it looks like it was kinda weird before this anyhow...Ah, well.